### PR TITLE
OSDOCS-12075 updating GCP disk types

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-12075

Link to docs preview:
https://82303--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installation-config-parameters-gcp.html#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@jianli-wei here is the main docs update for the GCP disk types. Please correct any mistakes I made with default values etc, thank you very much

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -2469,7 +2469,7 @@ When running your cluster on GCP 64-bit ARM infrastructures, ensure that you use
       osDisk:
         diskType:
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type] for compute machines.
-|`pd-ssd`, `pd-standard`, or `pd-balanced`. The default is `pd-ssd`.
+|Valid values are `pd-ssd`, `pd-standard`, `hyperdisk-balanced`, or `pd-balanced`. The default value is `pd-ssd`.
```
Comment (jianli-wei) [inline modules/installation-configuration-parameters.adoc:2472]: Suggest to list the valid types in the same order as the command `openshift-install explain`, thanks! 
```
$ openshift-install explain installconfig.compute.platform.gcp.osDisk.diskType
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "pd-balanced","pd-ssd","pd-standard","hyperdisk-balanced"
  DiskType defines the type of disk. For control plane nodes, the valid values are pd-balanced, pd-ssd, and hyperdisk-balanced.

$

**Review by jianli-wei — CHANGES_REQUESTED** [review]

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -2192,7 +2192,7 @@ When running your cluster on GCP 64-bit ARM infrastructures, ensure that you use
       osDisk:
         diskType:
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
-|The default disk type for all machines. Control plane nodes must use the `pd-ssd` disk type. Compute nodes can use the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
+|The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, `hyperdisk-balanced`. Control plane machines cannot use the `pd-standard` disk type, so if you specify it here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameter. The default value is `pd-ssd`.
```
Comment (jneczypor) [inline modules/installation-configuration-parameters.adoc:2195]:
```suggestion
|The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, or `hyperdisk-balanced`. The default value is `pd-ssd`. If you specify a value here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameters. Control plane machines cannot use the `pd-standard` disk type. 
```
I re-wrote the final sentences based on what I think you are trying to say. If it doesn't work, disregard my edits.

**Review by jneczypor — COMMENTED** [review]
A couple of changes, otherwise LGTM.
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/installation-configuration-parameters.adoc (hunk 2 above)
Comment (bscott-rh) [reply modules/installation-configuration-parameters.adoc:2195]: Thanks Janelle, I will reword slightly with your suggestion in mind.

**Pull request merged**